docs/openapi/components/schemas/common/AgGradeInspectionDefect.yml

```diff
@@ -0,0 +1,61 @@
+$linkedData:
+  term: AgGradeInspectionDefect
```

Comment:
I find the `Ag` abbreviation sorta annoying... from a convention perspective, I believe we should avoid abbreviations, as first time readers struggle with them... its an audience thing, not everyone will identify `Ag` as agriculture, especially non english speakers.

Reply:
That makes sense, though I have some reservations.

Do you think I should break from the convention established by AgActivity, AgProduct etc. and call these GradeInspectionDefect, or maybe AgricultureGradeInspectionDefect? Though, I'm concerned the first doesn't indicate that these are specific to agriculture, and the second seems pretty long (especially for something like AgricultureGradeInspectionSampleProperty).

Reply:
Whenever I see `Ag` my first reaction is "Silver"? Something like `AgricultureGradeInspectionDefect` would be a huge improvement.